src/main/scala/com/dimafeng/testcontainers/TestContainer.scala

```diff
@@ -16,32 +16,32 @@ import scala.concurrent.Future
 trait ForEachTestContainer extends SuiteMixin {
   self: Suite =>
 
-  val container: Container
```

Comment:
I think, this filed should stay and be annotated as deprecated to make it compatible with `override` in `override val container = ...`